tools/install_fedora_deps.sh

```diff
@@ -69,7 +70,7 @@ pushd "${tmp_dir}"
 git clone --depth=1 https://github.com/p4lang/behavioral-model
```

Comment:
Do we really need to build the behavioral model for the fedora tests? Can't we just run the normal p4 tests like we do with MacOS?